views/confirmation/confirmation.pug

```diff
@@ -7,6 +7,8 @@ block content
 
   h1 #{title}
 
+  include ../_includes/backButton
+
```

Comment:
I don't think we need to now (or maybe ever, but it looks like the following code snippet is copy+pasted on all o four pages.

```
  h1 #{title}

  include ../_includes/backButton
```

There's maybe an easy way to refactor this, but for right now let's not since it won't really give us that much.